code/game/machinery/computer/card.dm

```diff
@@ -395,6 +439,55 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 				carddesc += "<b>registered_name:</b> [target_owner]</span>"
 				jobs += "<b>Assignment:</b> [target_rank] (<a href='?src=[REF(src)];choice=demote'>Demote</a>)</span>"
 
+			var/banking = ""
+			banking += "<b>Department active & Bank account status:</b>"
+			banking += "<table border='1' cellspacing='1' cellpadding='0'>"
+			// Department active status
+			banking += "<tr>"
+			banking += "<td><b>Department manifest active:</b></td>"
```

Comment:
I don't even know what this is supposed to be to offer a better version

"Active Department Manifest"?